site/app/controllers/ManageSessionsController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace app\controllers;
+
+use app\libraries\Core;
+use app\libraries\response\WebResponse;
+use app\libraries\response\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * Class ManageSessionsController
+ * @package app\controllers
+ */
+class ManageSessionsController extends AbstractController {
+    /**
+     * ManageSessionsController constructor.
+     *
+     * @param Core $core
+     */
+    public function __construct(Core $core) {
+        parent::__construct($core);
+    }
+    /**
+     * Show manage sessions page
+     *
+     * @Route("/manage_sessions", methods={"GET"})
+     * @return WebResponse
+     */
+    public function showSessionsPage() {
+        return new WebResponse('ManageSessions', 'showSessionsPage');
+    }
+
+    /**
+     * Terminate a session
+     *
+     * @Route("/manage_sessions/terminate", methods={"POST"})
+     * @return JsonResponse
```

Comment:
For these various methods, what's the benefit to having this be a JsonResponse vs just having it be a regular redirect response back to the page?

Given that Submitty isn't a SPA, we shouldn't have AJAX around stuff that doesn't really need it as it requires additional JS code and such.

Reply:
There's no strong particular reason, I thought that AJAX enhances the user experience so that's why kept it this way....I'll change it to be a normal response.

Reply:
Given that I doubt any end user would particularly care or complain about using AJAX or regular HTTP post, I would opt for the path that uses the least amount of code, which in this case is without AJAX, allowing to remove the new JS file altogether.

Reply:
My motivation to prefer AJAX over regular POST comes from writings like this [ComputerWeekly article](https://www.computerweekly.com/news/2240081622/Use-Ajax-skills-to-enhance-the-online-user-experience). But given that Submitty isn't a leisure product but a necessity, it feels right to keep things simple.